src/badguy/dispenser.cpp

```diff
@@ -81,23 +81,23 @@ Dispenser::Dispenser(const ReaderMapping& reader) :
   set_colgroup_active(COLGROUP_MOVING_STATIC);
   SoundManager::current()->preload("sounds/squish.wav");
   reader.get("cycle", m_cycle, 5.0f);
-  if (reader.get("gravity", m_gravity)) m_physic.enable_gravity(true);
-  if ( !reader.get("badguy", m_badguys)) m_badguys.clear();
+  if (reader.get("gravity", m_gravity)) {
+    m_physic.enable_gravity(true);
+  }
+  if (!reader.get("badguy", m_badguys)) {
+    m_badguys.clear();
+  }
   reader.get("random", m_random, false);
   std::string type_s = "dropper"; //default
   reader.get("type", type_s, "");
-  try
-  {
+  try {
     m_type = DispenserType_from_string(type_s);
   }
-  catch(std::exception&)
-  {
-    if (!Editor::is_active())
-    {
+  catch(std::exception&) {
+    if (!Editor::is_active()) {
```

Comment:
The coding style document states that brackets should stand on their own line when possible; it would be the code that doesn't do that already that should be changed. I'll still review with the rest of the team which style we should favor.

Reply:
K&R vs Allman... the classic holy wars
_Fight!_

Reply:
In the Linux kernel coding style fewer lines are preferred.
https://01.org/linuxgraphics/gfx-docs/drm/process/coding-style.html#placing-braces-and-spaces

> Also, note that this brace-placement also minimizes the number of empty (or almost empty) lines, without any loss of readability. Thus, as the supply of new-lines on your screen is not a renewable resource (think 25-line terminal screens here), you have more empty lines to put comments on.